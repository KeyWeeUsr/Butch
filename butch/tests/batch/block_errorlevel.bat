@echo off

:: default status is 0
echo %errorlevel%

:: single wrapped command
(
    ver > nul
)
:: zero
echo %errorlevel%

(
    :: zero
    ver > nul
    :: non-zero
    type nonexisting
)
:: non-zero
echo %errorlevel%

(
    :: non-zero
    type nonexisting
    :: zero
    ver > nul
)
:: zero
echo %errorlevel%
