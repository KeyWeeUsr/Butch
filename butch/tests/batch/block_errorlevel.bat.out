0
<null>
0
<null>
The system cannot find the file specified.
1
The system cannot find the file specified.
<null>
0
